test11.java

```diff
@@ -0,0 +1,15 @@
+class ThrowInFinally {
+    @Test
+    void test_1() {
+        try {
+        } finally {
+            throw new RuntimeException();
+            return;
```

Comment:
RSPEC-1143: Don't throw or return inside a finally (line 13)
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._